pages/common/dc.md

```diff
@@ -0,0 +1,23 @@
+# dc
+
+> An arbitrary precision calculator. Uses reverse polish notation (RPN).
+
+- Run calculator in interactive mode:
+
+`dc`
+
+- Execute dc script in file:
+
+`dc -f {{file}}`
+
+- Calculate 4 times 5 [4 5 *], subtract 17 [17 -], and [p]rint the output (using echo):
+
+`echo "4 5 * 17 - p"| dc`
+
+- Set number of decimal places to 10 [k 7], calculate 5 divided by -3 [5 _3 /] and [p]rint (using dc -e):
```

Comment:
Shouldn't this be [10 k] ? Considering you have [100 k] in the next example ?

Reply:
Sigh. Yes. Errant vim keystrokes on home laptop. I'll use 7 places since the code actually does too.